jupyter-notebooks/orders_basemaps/SDK_order_basemaps.ipynb

```diff
@@ -0,0 +1,497 @@
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Use Basemaps API to get the Basemap and quad IDs\n",
+    "---\n",
+    "\n",
+    "### What is a Quad ID? \n",
+    "\n",
+    "A quad ID is a unique identifer to access a square tile of imagery of the Earth's surface, whereas a Basemap is a combination of squares. \n",
+    "\n",
+    "Use the Basemap APIs to retrieve the basemap you want and the quad IDs you are looking for. You can get the base URL you'll need to communicate with the Basemap API service:\n",
+    "\n",
+    "1. Go to the [Basemap API reference](https://developers.planet.com/docs/basemaps/reference/) at https://developers.planet.com/docs/basemaps/reference/. \n",
+    "2. Under List Mosaics, select GET /mosaics.\n",
+    "\n",
+    "A dropdown UI element appears with the URL to use:\n",
+    "\n",
+    "https://api.planet.com/basemaps/v1/mosaics\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import requests\n",
+    "from requests.auth import HTTPBasicAuth\n",
+    "import json\n",
+    "import math"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "auth = HTTPBasicAuth(PL_API_KEY, '')\n",
+    "session = requests.Session()\n",
+    "\n",
+    "# Authenticate\n",
+    "session.auth = (PL_API_KEY, '')"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Search for the mosaic ID to get its quad IDs\n",
+    "\n",
+    "Here, we are using the full name of the mosaic, but with the `name__contains` field, you can use a partial name based on the names of the mosaics available through your account.\n",
+    "\n",
+    "Declare the search string for `requests.get()`. You will use the dictionary you create with the mosaic name to pass to the requests.get() function.\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This code sets the URL and headers for an HTTP request to Planet's Basemap API and sends a GET request with the specified search parameters. If the response returns the correct status, the JSON response is loaded into a Python dictionary with mosaics information from which we'll extract quad ID information."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "BASEMAP_API_URL = 'https://api.planet.com/basemaps/v1/mosaics'\n",
+    "headers = {'content-type': 'application/json'}\n",
+    "\n",
+    "basemap_params = {\n",
+    "    'name__contains': 'point_reyes_rolling_normalized_2020-04-30_mosaic'\n",
+    "}\n",
+    "\n",
+    "basemap_service_response = requests.get(\n",
+    "    url=BASEMAP_API_URL, params=basemap_params, auth=auth, headers=headers)\n",
```

Comment:
Why'd you set up a session if you're not going to use it? Either don't make the session, or use
```
basemap_service_response = session.get(url=BASEMAP_API_URL, params=basemap_params, headers=headers)
```
You could probably put the headers in the session too, instead of passing them here. e.g.:
```
session.headers.update({'content-type': 'application/json'})
```